src/Groups/homomorphisms.jl

```diff
@@ -118,12 +118,12 @@ If `check` is set to `false` then it is not checked whether the mapping
 defines a group homomorphism.
 """
 function hom(G::GAPGroup, H::GAPGroup, gensG::Vector, imgs::Vector; check::Bool = true)
-  vgens = GapObj([x.X for x in gensG])
-  vimgs = GapObj([x.X for x in imgs])
+  vgens = GapObj([GapObj(x) for x in gensG])
+  vimgs = GapObj([GapObj(x) for x in imgs])
```

Comment:
Here's another place one could now just do a recursive conversion:
```suggestion
  vgens = GapObj(gensG; recursive=true)
  vimgs = GapObj(imgs; recursive=true)
```